File: text/haskell/delete/text_delete.hs
#! /usr/bin/runghc
-- -----------------------------------------------------------------
--	text_delete.hs
--
--						Dec/26/2014
--
-- -----------------------------------------------------------------
import	System.Environment
import	Data.Map
import	System.IO.Strict

import Text_manipulate
-- -----------------------------------------------------------------
main :: IO ()
main = do
	putStrLn "*** 開始 ***"
	args <- getArgs
	let file_name = head args
	let key_in = head (tail args)
	putStr (key_in ++ "\n")
--
	inStr <- System.IO.Strict.readFile file_name
	let dict_aa = str_to_dict_proc inStr
--
	let dict_bb = delete key_in dict_aa
--
	let str_aa = dict_to_str_proc "\t" dict_bb
	writeFile file_name str_aa
--
	putStrLn "*** 終了***"
-- -----------------------------------------------------------------

File: text/julia/delete/text_delete.jl
#! /usr/bin/julia
#
#	text_delete.jl
#
#					Jul/05/2018
# --------------------------------------------------------------------
include("text_manipulate.jl")
#
# --------------------------------------------------------------------
println("*** 開始 ***")
fname_text = ARGS[1]
key_in = ARGS[2]
println(fname_text)
println(key_in)
#
dict_aa = text_to_dict_proc(fname_text)
#
if ( haskey(dict_aa,key_in))
	delete!(dict_aa, key_in)
	dict_to_text_proc(dict_aa,fname_text)
end
#
println("*** 終了 ***")
# --------------------------------------------------------------------
